docs/inference_api.md

```diff
@@ -69,7 +69,7 @@ or:
 
 import requests
 
-res = requests.post("http://localhost:8080/predictions/squeezenet1_1", files={'data': open('docs/images/dogs-before.jpg', 'rb'), 'data': open('docs/images/kitten_small.jpg', 'rb')})
+res = requests.post("http://localhost:8080/predictions/squeezenet1_1", files=[('data', open('docs/images/dogs-before.jpg', 'rb')), ('data', open('docs/images/kitten_small.jpg', 'rb'))])
```

Comment:
This sounds like it doesn't quite work based on your other PR? In case it does I'd push it in its own mini section on sending multiple images in a single inference request

Reply:
Yes, it doesn't work to me. I'm going to research why tomorrow.

Reply:
How about we revert the change here so we can merge the rest of your PR?

Reply:
Ok, I will do it today, 
Actually after some research I didn't understand how to fix this part&